core/src/com/unciv/logic/map/mapunit/MapUnit.kt

```diff
@@ -486,6 +486,11 @@ class MapUnit : IsPartOfGameInfoSerialization {
             damageFactor *= 1f - unique.params[0].toFloat() / 100f
         return damageFactor
     }
+    
+    fun airSweepDamagePercentBonus(): Int {
```

Comment:
This is only used in one place, it really doesn't need to be in a centralized place like this